tests/test_codebase/test_mmpose/data/yolox-pose_s_8xb32-300e_coco.py

```diff
@@ -0,0 +1,259 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
@huayuan4396 may remove unused params in this model config, you can refer to this ut model config:
https://github.com/open-mmlab/mmdeploy/blob/main/tests/test_codebase/test_mmpose/data/model.py